src/client.js

```diff
@@ -54,6 +56,7 @@ function createMethod(clientMethod, dbg, cancelCache) {
     Subject / vs ReplaySubject might miss some entities!
     */
     const retObs = new Observable(observer => {
+      __test__.current = observer;
```

Comment:
TODO remove, this is just a basic way to ensure you are working the correct observable in the tests via equality check. In most cases this should be the observable returned from `.subscribe()`, but it might be a child if you used take, etc.